### PR TITLE
Fix humphd/brackets#190 - Add a button to toggle between an attached and detached live preview state

### PR DESCRIPTION
**Pull request by AJDBenner**
Fixes https://github.com/humphd/brackets/issues/190. This PR adds a button for the live preview. It's a WIP and still requires the function call to open the detached live preview window. Since we're no longer just hiding UI preferences we are altering the UI, I changed the `hideUI.js` file to `alterUI.js` and the `hide()` function to `alter()`. 

The change looks like this

screen shot 2015-04-14 at 2 23 03 pm

### CODE REVIEW

File: lib/alterUI.js [hunk 1]
```diff
@@ -86,8 +87,19 @@ define(function (require, exports, module) {
         $(".content").css("right","0");
     }
 
+    /**
+     * Used to add a button to the status bar for the
+     * detached live preview.
+     */
+    function addLivePreviewButton() {
+        $("#status-indicators").append("<button id=liveDevButton>Live Preview</button>");
```
Comment (humphd) [inline lib/alterUI.js]: I really think this needs to be an icon, not a button with that much text.  Can you look at other Brackets extensions that do similar things and just do an icon or img?

File: lib/alterUI.js [hunk 2]
```diff
@@ -12,12 +12,13 @@ define(function (require, exports, module) {
      * This function calls all the hide functions and listens
      * for bramble to be loaded
      */
-    function hide() {
+    function alter() {
```
Comment (humphd) [inline lib/alterUI.js]: `initUI()` I think.

File: main.js [hunk 3]
```diff
@@ -23,7 +23,7 @@ define(function (require, exports, module) {
         Editor               = brackets.getModule("editor/Editor").Editor,
         // Load nohost dependencies
         Browser              = require("lib/iframe-browser"),
-        HideUI               = require("lib/hideUI"),
+        AlterUI              = require("lib/alterUI"),
```
Comment (humphd) [inline main.js]: I'd just call it `UI.js` since it's our custom UI code for Bramble

**Comment by AJDBenner** [comment]
Closing this PR because it was created against the wrong repo. The new PR in the proper repo can be found [here](https://github.com/AJDBenner/brackets-browser-livedev/pull/3)

**Pull request closed without merging**